### PR TITLE
LinwoodCloud.Butterfly.Nightly version 1.3.0-rc.0

### PR DESCRIPTION
**Pull request by CodeDoctorDE**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/60487)

### CODE REVIEW

**Comment by CodeDoctorDE** [comment]
discord.linwood.dev works perfect?

File: manifests/l/LinwoodCloud/Butterfly/Nightly/1.3.0-rc.0/LinwoodCloud.Butterfly.Nightly.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+PackageIdentifier: LinwoodCloud.Butterfly.Nightly
+PackageVersion: 1.3.0-rc.0
+PackageLocale: en-US
+Publisher: LinwoodCloud
+PublisherUrl: https://linwood.dev
+PublisherSupportUrl: https://go.linwood.dev/discord
+PrivacyUrl: https://docs.butterfly.linwood.dev/privacypolicy
+Author: LinwoodCloud
+PackageName: Butterfly
```
Comment (ItzLevvie) [inline manifests/l/LinwoodCloud/Butterfly/Nightly/1.3.0-rc.0/LinwoodCloud.Butterfly.Nightly.locale.en-US.yaml:12]:
```suggestion
PackageName: Butterfly Nightly
```

File: manifests/l/LinwoodCloud/Butterfly/Nightly/1.3.0-rc.0/LinwoodCloud.Butterfly.Nightly.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+PackageUrl: https://docs.butterfly.linwood.dev
+License: GPL-3.0
+LicenseUrl: https://github.com/LinwoodCloud/Butterfly/blob/develop/LICENSE
+Copyright: LinwoodCloud
+CopyrightUrl: https://github.com/LinwoodCloud/Butterfly
+ShortDescription: Nightly version of the note-taking app Butterfly
+Description: Nightly version of the note-taking app Butterfly
+Moniker: Note
```
Comment (ItzLevvie) [inline manifests/l/LinwoodCloud/Butterfly/Nightly/1.3.0-rc.0/LinwoodCloud.Butterfly.Nightly.locale.en-US.yaml:20]:
```suggestion
Moniker: butterfly-nightly
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Merged in https://github.com/microsoft/winget-pkgs/commit/b2f964dc3f368dfb180c23bbc06fec70dc864772

image

**Pull request closed without merging**